src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -73,6 +73,36 @@ public static void ResetColor()
             }
         }
 
+        public static int WindowWidth
+        {
+            get
+            {
+                if (!ConsoleOutIsTerminal) return 0; // Should we rather throw exception here? On windows we get IOException from the native code, do we need to match for code portability?
+                return TerminalBasicInfo.Instance.ColumnFormat;
```

Comment:
I don't think this works.  The terminfo database is static; it doesn't change based on the current state of the windows.  This is going to just return the default value of 80.